--- java-1.7.0-openjdk/openjdk/jdk/src/share/native/sun/awt/splashscreen/splashscreen_gif.c.bero	2013-02-28 00:35:47.790222526 +0100
+++ java-1.7.0-openjdk/openjdk/jdk/src/share/native/sun/awt/splashscreen/splashscreen_gif.c	2013-02-28 00:40:14.333890271 +0100
@@ -316,7 +316,7 @@ SplashDecodeGif(Splash * splash, GifFile
 int
 SplashDecodeGifStream(Splash * splash, SplashStream * stream)
 {
-#ifdef GIFLIB_MAJOR >= 5
+#if GIFLIB_MAJOR >= 5
     int error = 0;
     GifFileType *gif = DGifOpen((void *) stream, SplashStreamGifInputFunc, &error);
 
